File: 2_Kernel/JavaKernel/src/main/java/com/seagosoft/tws/data/BondContractDetailsData.java
package com.seagosoft.tws.data;

import com.ib.client.ContractDetails;

public class BondContractDetailsData {

    public int reqId;
    public ContractDetails contractDetails;

    public BondContractDetailsData(int reqId, ContractDetails contractDetails) {
        this.reqId = reqId;
        this.contractDetails = contractDetails;
    }
}

File: 3_OrderSystem/KotlinEWrapperBackend/src/main/kotlin/com/seagosoft/tws/wrapper/structs/BondContractDetailsData.kt
package com.seagosoft.tws.wrapper.structs

import com.ib.client.ContractDetails

class BondContractDetailsData(var reqId: Int, var contractDetails: ContractDetails) {
    override fun toString(): String {
        return "BondContractDetailsData(reqId=$reqId, contractDetails=$contractDetails)"
    }
}